devdoc/doublylinkedlist_requirements.md

```diff
@@ -104,3 +134,48 @@ extern PDLIST_ENTRY DList_RemoveHeadList(PDLIST_ENTRY listHead);
 **SRS_DLIST_06_013: [** DList_RemoveHeadList shall return listHead if that's the only item in the list. **]**
 
 Note: The Flink & Blink of the returned PDLIST_ENTRY shall be undefined.
+
+### DList_Find
+```c
+MOCKABLE_FUNCTION(, DLIST_FIND_RESULT, DList_Find, PDLIST_ENTRY, listHead, DLIST_MATCH_FUNCTION, matchFunction, const void*, matchContext, PDLIST_ENTRY*, foundEntry);
+```
+
```

Comment:
traditionally we have one phrase that explains what the function does in non-requirement terms here.

Reply:
Here and everywhere so I don't repeat myself.

Reply:
Added description for DList_ForEach in next iteration.